<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalPropertiesWidget</class>
 <widget class="QWidget" name="GlobalPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>Dashboard Properties</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold; font-size: 14px; padding: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="appearance_group">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QFormLayout" name="appearance_layout">
      <item row="0" column="0">
       <widget class="QLabel" name="background_label">
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="background_color_button">
        <property name="text">
         <string>Choose Color</string>
        </property>
        <property name="styleSheet">
         <string>background-color: rgb(240, 240, 240);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="show_grid_checkbox">
        <property name="text">
         <string>Show Grid</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="behavior_group">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="behavior_layout">
      <item>
       <widget class="QCheckBox" name="snap_to_grid_checkbox">
        <property name="text">
         <string>Snap to Grid</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 